### PR TITLE
Provide description for measurement transcript usage.

### PR DESCRIPTION
**Pull request by jyao1**

### CODE REVIEW

File: src/08-attestation.adoc [hunk 1]
```diff
@@ -203,6 +203,35 @@ Although the device measurement and certificate are not required to be included
 in the TVM report, the TVM should provide a mechanism to return the device
 measurement and certificate for the verifier to perform further verification.
 
+To support the remote verification, the device measurement data should be the
```
[the remark this answers is not in the document]
Comment (jyao1) [reply src/08-attestation.adoc]: Fixed

File: src/08-attestation.adoc [hunk 2]
```diff
@@ -203,6 +203,35 @@ Although the device measurement and certificate are not required to be included
...
+signed <<SPDM>> measurement transcript, including `VCA` and all
+`{GET_MEASUREMENTS, MEASUREMENTS}` pairs that are exchanged between the SPDM
+measurement requester and the responder. Only the last `MEASUREMENTS` shall
+include the digital signature of the measurement transcript.
+
+To provide the signed <<SPDM>> measurement transcript has multiple benefits:
```
[the remark this answers is not in the document]
Comment (jyao1) [reply src/08-attestation.adoc]: Fixed

File: src/08-attestation.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (jyao1) [reply src/08-attestation.adoc]: Yes. Changed.

**Review by sameo — APPROVED** [review]

File: src/08-attestation.adoc [hunk 3]
```diff
@@ -203,6 +203,34 @@ Although the device measurement and certificate are not required to be included
 in the TVM report, the TVM should provide a mechanism to return the device
 measurement and certificate for the verifier to perform further verification.
 
+To support remote verification, the device measurement data shall the
```
Comment (sameo) [inline src/08-attestation.adoc:206]:
```suggestion
To support remote verification, the device measurement data shall be the
```

Comment (jyao1) [reply src/08-attestation.adoc]: fixed

**Pull request merged**